modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCActionCommand.java

```diff
@@ -60,14 +61,10 @@ protected void doProcessAction(
 		}
 	}
 
-	private void _checkDestinationRepository(long repositoryId)
-		throws PortalException {
-
-		Group group = _groupLocalService.fetchGroup(repositoryId);
-
+	private void _checkDestinationGroup(Group group) throws PortalException {
 		if ((group != null) && group.isStaged() && !group.isStagingGroup()) {
 			throw new PortalException(
-				"cannot-copy-file-entries-to-the-live-version-of-a-group");
+				"cannot-copy-folders-to-the-live-version-of-a-group");
```

Comment:
Is this key change correct? Looks like a typo to me.

Reply:
oh! 
Thanks!

I will rechange it!!